cdap-data-fabric/src/main/java/co/cask/cdap/data/runtime/DataSetsModules.java

```diff
@@ -119,7 +119,7 @@ protected void configure() {
 
   @Override
   public Module getDistributedModules() {
-    return getDistributedModules(false);
+    return getDistributedModules(true);
```

Comment:
this will change default behavior right? I mean this will bind ``KafkaMetadataPublisher`` every place where ``getDistributedModules()`` is called. Might cause unit test failures?

Perhaps it would be better to find where publishing is required, and pass a ``true`` from there?